src/main/java/org/sagebionetworks/web/client/widget/doi/CreateOrUpdateDoiModal.java

```diff
@@ -55,6 +56,8 @@
 	private PopupUtilsView popupUtilsView;
 	private DateTimeUtils dateTimeUtils;
 
+	private boolean doiExists = false;
```

Comment:
Moved this to a class variable so we can access it outside of the configure method